integration/local-gadget/Makefile

```
@@ -0,0 +1,70 @@
+CONTAINER_CLI ?= docker
+MINIKUBE_VERSION ?= v1.27.0
+
+CONTAINER_RUNTIME ?= docker
+
+# make does not allow implicit rules (with '%') to be phony so let's use
+# the 'phony_explicit' dependency to make implicit rules inherit the phony
+# attribute
+.PHONY: phony_explicit
+phony_explicit:
+
+# minikube
+
+MINIKUBE = $(shell pwd)/bin/minikube-$(MINIKUBE_VERSION)
+.PHONY: minikube
+minikube:
+	mkdir -p bin
+	test -e bin/minikube-$(MINIKUBE_VERSION) || \
+	(cd bin && curl -Lo ./minikube-$(MINIKUBE_VERSION) https://github.com/kubernetes/minikube/releases/download/$(MINIKUBE_VERSION)/minikube-linux-amd64)
+	chmod +x bin/minikube-$(MINIKUBE_VERSION)
+
+# clean
+
+.PHONY: clean
+clean: minikube
+	$(MINIKUBE) delete -p minikube-docker
+	$(MINIKUBE) delete -p minikube-containerd
+	$(MINIKUBE) delete -p minikube-cri-o
+
+# setup
+
+SETUP_TARGETS = \
+	setup-docker \
+	setup-containerd \
+	setup-cri-o
+
+.PHONY: setup-all
+setup-all: $(SETUP_TARGETS) setup
+
+setup: setup-$(CONTAINER_RUNTIME)
+
+.PHONY: phony_explicit
+setup-%: minikube
+	$(CONTAINER_CLI) inspect minikube-$* >/dev/null 2>&1 || \
+	$(MINIKUBE) start -p minikube-$* --driver=docker --container-runtime=$* --wait=all
+
+# test
+
+TEST_TARGETS = \
+	test-docker \
+	test-containerd \
+	test-cri-o
+
+.PHONY: test-all
+test-all: $(TEST_TARGETS) test
+
+test: test-$(CONTAINER_RUNTIME)
+
+.PHONY: phony_explicit
+test-%:
+	export MINIKUBE_PROFILE=minikube-$* && \
+	echo "Checking minikube with profile $${MINIKUBE_PROFILE} is running ..." && \
+	$(CONTAINER_CLI) inspect $${MINIKUBE_PROFILE} >/dev/null 2>&1 || (echo "Error: $${MINIKUBE_PROFILE} not running, exiting ..." && exit 1) && \
+	echo "Preparing minikube with profile $${MINIKUBE_PROFILE} for testing ..." && \
+	$(CONTAINER_CLI) cp ../../local-gadget-linux-amd64 $${MINIKUBE_PROFILE}:/bin/local-gadget >/dev/null 2>&1 && \
+	go test -c -o local-gadget-integration.test ./... && \
+	$(CONTAINER_CLI) cp local-gadget-integration.test $${MINIKUBE_PROFILE}:/bin/ >/dev/null 2>&1 && \
+	rm local-gadget-integration.test && \
+	echo "Running test in minikube with profile $${MINIKUBE_PROFILE} ..." && \
+	$(CONTAINER_CLI) exec $${MINIKUBE_PROFILE} local-gadget-integration.test -integration -container-runtime $*
```

I propose adding `-test.v` in this initial phase, we can remove it once the tests are more stable:

```suggestion
	$(CONTAINER_CLI) exec $${MINIKUBE_PROFILE} local-gadget-integration.test -integration -test.v -container-runtime $*
```